src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+// Assuming other class definitions remain the same
```

Comment:
Overall your Duke class looks neat and tidy. No excessively long codes or very deep nested conditionals. Nothing much to comment but perhaps you can push your other class files to github and I can take a look and add more comments there!